src/node-opendds.cpp

```diff
@@ -96,6 +101,7 @@ namespace {
     ot->SetInternalFieldCount(1);
     Nan::SetMethod(ot, "subscribe", subscribe);
     Nan::SetMethod(ot, "unsubscribe", unsubscribe);
+    Nan::SetMethod(ot, "unsubscribe_now", unsubscribe_now);
```

Comment:
Do we expect user code to need to invoke this, or is it just here to make it work with setImmediate?